package.js

```diff
@@ -1,18 +1,20 @@
 Package.describe({
-	summary: "Javascript forms validation. Powerful, UX aware, and dead simple. Internationalised.",
-	version: "2.0.1",
-	git: "https://github.com/ephemer/meteor-parsley"
+    name: 'ephemer:parsley',
+    summary: 'Javascript forms validation. Powerful, UX aware, and dead simple. Internationalised.',
+    version: '2.3.11',
+    git: 'https://github.com/ephemer/meteor-parsley'
 });
 
 Package.onUse(function(api) {
-	api.versionsFrom('METEOR@0.9.0');
-  	api.use(['jquery'], 'client');
+    api.versionsFrom('1.2');
+    api.use('ecmascript');
+    api.use(['jquery'], 'client');
 
     // Parsley
-    api.add_files(['parsley.js'], 'client');
+    api.addFiles(['parsley.min.js'], 'client', {bare: true});
```

Comment:
lieber nicht `.min` nutzen – das macht das Debugger schwierig und non-minified Files werden in Production trotzdem durch Meteor minified